clarity/src/vm/ast/parser/v2/lexer/mod.rs

```diff
@@ -848,7 +860,13 @@ impl<'a> Lexer<'a> {
             | Token::Less
             | Token::LessEqual
             | Token::Greater
-            | Token::GreaterEqual => {
+            | Token::GreaterEqual
+            | Token::BitwiseXor
```

Comment:
So, this is one of the reasons why I'm not a fan of having two tokens for xor -- you have to remember to use both in all `match` cases, which is a consensus-critical bug.  Seems you're missing `BitwiseXor2`.